nifi-registry/nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/security/authentication/jwt/JwtService.java

```diff
@@ -34,11 +34,13 @@
 import org.apache.nifi.registry.security.key.KeyService;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
 import org.springframework.stereotype.Service;
 
 import java.nio.charset.StandardCharsets;
-import java.util.Calendar;
+import java.util.*;
```

Comment:
Coding conventions require explicit imports, so this change should be reverted.

Reply:
Sorry, IntelliJ insists on reverting this every time I update it. Fixed.